tests/php/test-tag-and-attribute-sanitizer.php

```diff
@@ -787,18 +788,20 @@ static function() {
 				[ 'amp-sidebar' ],
 			],
 
-			'amp_sidebar_with_div_inside_nav'              => [
+			'amp_sidebar_in_div_with_div_inside_nav'       => [
```

Comment:
Was this meant to *replace* the previous data set? Or should it have been an *addition*?